File: ЭКЗАМЕН/3.6.go
package main

import(
	"fmt"
)

func main(){
	var n int
	fmt.Println("Введите чмсло: ")
	fmt.Scan(&n)

	num := make([]int, n)
	fmt.Println("введите числа ")
	for i := 0; i < n; i++ {
		fmt.Scan(&num[i])
	} 
	result := 0

	for _, num := range num{
		result ^=  num
	}
	fmt.Println("Числа без пары ")

}

File: ЭКЗАМЕН/3.6.cpp
#include <iostream>
#include <cmath>
#include <vector>
using namespace std;

int main(){
    setlocale(LC_ALL, "RU");
    int n;
    cout << "введите число: " << endl;
    cin >> n;

    //Создадим массив
    vector<int> number(n);
    cout << " Введите число: ";
    for(int i = 0; i < n; i++){
        cin >> number[i];
    }
    int result = 0;
    for(int num : number){
        result ^= num;
    }
    cout << "Ответ: " << result << endl;
    return 0;


}